components/gitpod-cli/cmd/forward-port.go

```diff
@@ -26,7 +27,7 @@ var portFwdCmd = &cobra.Command{
 	Long:  ``,
 	Args:  cobra.RangeArgs(1, 2),
 	Run: func(cmd *cobra.Command, args []string) {
-		srcp, err := strconv.ParseInt(args[0], 10, 32)
+		srcp, err := num.ParseInt32(args[0])
```

Comment:
I'm not sure I follow the benefit of this change. The `strconv.ParseInt` would produce about the same error, no?

Reply:
fixed